Makefile

```diff
@@ -2,8 +2,10 @@
 
 # Start from clean env: Delete `.venv`, then `python3 -m venv .venv`
 # Pre-requisite: Python virtual environment is active (source .venv/bin/activate)
-build-python: release
-  python3 -m venv .venv
-  source .venv/bin/activate
-  python3 -m pip install -r requirements.txt
-  maturin develop
+
+build-python:
+	rm -rf c2pa/c2pa
```

Comment:
We really need to make sure about the version match. Hence this takes care of uninstalling the found maturin/uniffi and reinstalling the version we expect from `requirements.txt`.